src/test/java/seedu/address/model/game/FlatModeTest.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.model.game;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.address.model.task.Status;
+import seedu.address.model.task.Task;
+import seedu.address.testutil.TaskBuilder;
+
+// @@author chikchengyao
+
+public class FlatModeTest {
+    private GameManager gm;
+
+    private int overdueXp = 2347;
+    private int completedXp = 9362;
+
+    @Before
+    public void setUp() {
+        gm = new GameManager(new FlatMode(overdueXp, completedXp));
+    }
+
+    @Test
+    public void constructorTest() {
+        FlatMode fm1 = new FlatMode();
+        FlatMode fm2 = new FlatMode(50, 100);
+    }
+
+    @Test
+    public void null_appraiseXpChange() {
```

Comment:
the test name should be in the format of `<feature under test>_<test scenario>_<expected outcome>`
https://github.com/oss-generic/process/blob/master/codingStandards/CodingStandard-Java.adoc